.vscode/cspell.json

```diff
@@ -74,6 +75,7 @@
     "mongodb",
     "mysql",
     "ndjson",
+    "netlify",
```

Comment:
Please also add `Netlify` to `.textlintrc`.

Reply:
We should also add a rule for GitPod, but I can do that after this is merged if you prefer (we'll need to allow for gitpod.io).

Reply:
Done, I also added GitPod, didn't have any issues with the URLs